README.md

```diff
@@ -0,0 +1,6 @@
+sonarqube role
+=========
+[![License](https://img.shields.io/badge/license-Apache-green.svg?style=flat)](https://raw.githubusercontent.com/lean-delivery/ansible-role-sonarqube/master/LICENSE)
+[![Build Status](https://travis-ci.org/lean-delivery/ansible-role-sonarqube.svg?branch=master)](https://travis-ci.org/lean-delivery/ansible-role-sonarqube)
+
+## Summary
```

Comment:
Please add description of role variables and few examples of usage

Reply:
In progress...